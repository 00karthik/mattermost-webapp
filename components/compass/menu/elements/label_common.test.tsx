// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
// See LICENSE.txt for license information.

import React from 'react';
import {shallow} from 'enzyme';

import LabelCommon from './label_common';

describe('components/compass/menu/LabelCommon', () => {
    test('should match snapshot', () => {
        const Props = {
            children: 'text',
            size: 'small',
        };

        const wrapper = shallow(
            <LabelCommon
                {...Props}
            />,
        );

        expect(wrapper).toMatchSnapshot();
        expect(wrapper.find('span').props().className).toBe('Label Label___small');
    });
});