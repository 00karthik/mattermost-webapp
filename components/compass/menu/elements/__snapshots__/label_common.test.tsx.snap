// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/compass/menu/LabelCommon should match snapshot 1`] = `
<div
  className="Label__wrapper"
>
  <span
    className="Label Label___small"
  >
    text
  </span>
</div>
`;
